### ISSUE TITLE
The new background color for the "Highlight spoken text" option causes a color conflict with some high-contrast color modes in Windows 10

### ISSUE THREAD

**Issue by k-kolev1985** [blindpandas/bookworm#64]
## Describe the Problem

If the "High-Contrast Black" theme is used in Windows 10, the new yellow background color for the option "Highlight spoken text" causes a color conflict in Bookworm. The reason is that with the "High-Contrast Black" theme of Windows 10, the color of the text is white. Combined with the yellow background color for the highlighted text, this causes the white highlighted text to become very hard to read. A solution would be to change the color of the highlighted text as well, making it for example black - black text on a yellow background is OK to read.

## To Reproduce

1. Enable high-contrast mode in Windows 10, by doing the following:
1.1. Open Windows Settings (Windows+I).
1.2. Go to Ease of access -> High-contrast.
1.3. Turn on the high-contrast mode, using the respective toggle.
1.4. From the "High-contrast theme" combo-box, select "High-Contrast Black".
2. Launch Bookworm.
3. Open the preferences of Bookworm.
4. From the "Reading" category, enable the option "Highlight spoken text" (it should be enabled by default, but check it out, just in case).
5. Press OK to apply and save the changes.
6. Open a file in Bookworm.
7. Press F5 to start the reading process.
8. Observe how the highlighted in yellow white text is hard (if not impossible) to read.

## Expected behavior

The highlighted text should be in a color, that does not conflict (has a good contrast) with the color of the highlight background color.

## The File You've Opened

The file format does not matter in this case.

## Desktop (please complete the following information):

 - OS: Windows 10 Pro version 20H2 (build 19042.985), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
 - Bookworm version: 0.4b1.

**Comment by DraganRatkovich** [comment]
Hello,
@k-kolev1985 Could you test the latest build of Bookworm, is it fixed or not?

**Comment by k-kolev1985** [comment]
@DraganRatkovich 
Yes, the problem still persists. Version of Bookworm is 2022.1a5. Is there a newer build?

**Comment by cary-rowen** [comment]
You can download the latest build from [this link](https://ci.appveyor.com/api/buildjobs/ykk1m99oquvq1dn2/artifacts/scripts%2FBookworm-2022.1b1-x64-setup.exe)
btw, @mush42 You might consider releasing a new version.

**Comment by k-kolev1985** [comment]
I tried this build, but couldn't get far with it - it kept crashing after I loaded a file in it or invoked the program's preferences dialog.
